copy.client.ts

```diff
@@ -1,10 +1,31 @@
-const copyBtns = document.querySelectorAll("button[data-copy]");
-
-copyBtns.forEach((btn) => {
-  btn.addEventListener("click", () => {
-    let textToCopy = btn.getAttribute("data-copy") as string;
-    // CLEAN COMMANDS:  Remove leading spaces, $, and > from each line
-    textToCopy = textToCopy.replace(/^[\$>\s]+/, "");
-    navigator?.clipboard?.writeText(textToCopy);
+document.addEventListener("click", (event) => {
+  const btn = (event.target as HTMLElement).closest("button[data-copy]");
+
+  if (!btn) {
+    return;
+  }
+
+  let textToCopy = btn.getAttribute("data-copy") as string;
+
+  // CLEAN COMMANDS:  Remove leading spaces, $, and > from each line
+  textToCopy = textToCopy.replace(/^[\$>\s]+/, "");
+
+  navigator?.clipboard?.writeText(textToCopy).then(() => {
+    if (!btn) {
+      return;
+    }
```

Comment:
nit: I don't think this can be null or undefined even if the button was removed at this point